iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="backdrop" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" role="none" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
Maybe `role="presentation"`? http://www.w3.org/TR/wai-aria/roles#presentation `aria-hidden="true"` seems to indicate that the thing its applied to isn't visible at all.